src/AlgebraicGeometry/Surfaces/EnriquesAut.jl

```diff
@@ -0,0 +1,997 @@
+  t2 = change_base_ring(GF(2), D2.tau)
+  for (i,g2) in enumerate(initial_automorphisms_mod2(D1.data))
+    h2 = t1*g2*t2
+    if D1.data.membership_test(D1.data, h2)
+      g = initial_automorphisms(D1.data)[i]
+      h = tau1inv*g*D2.tau
+      push!(result, h)
+    end
+  end
+  return result
+end
+
+@doc raw"""
+    is_congruent_with_data(D1::EnriquesChamber, D2::EnriquesChamber) -> Bool, ZZMatrix
+    
+Return whether `D1` and `D2` are congruent under ``G_Y^0`` and if yes
+an element ``g \in G_Y^0`` with ``D1^g=D2``.
+"""
+function is_congruent_with_data(D1::EnriquesChamber, D2::EnriquesChamber)
+  tau1inv = inv(D1.tau)
+  t1 = change_base_ring(GF(2),tau1inv)
+  t2 = change_base_ring(GF(2), D2.tau)
+  for (i,g2) in enumerate(initial_automorphisms_mod2(D1.data))
+    h2 = t1*g2*t2
+    if D1.data.membership_test(D1.data, h2)
+      g = initial_automorphisms(D1.data)[i]
+      h = tau1inv*g*D2.tau
+      return true, h
+    end
+  end
+  return false, tau1inv
+end
+
+aut(D::EnriquesChamber) = hom(D, D)
+
+@doc raw"""
+    borcherds_method(Y::EnriquesBorcherdsCtx; max_nchambers=-1)
+    
+Compute ``\mathrm{Aut}_{s}(Y)`` of the Enriques surface ``Y`` using Borcherds method. 
+
+Here ``\mathrm{Aut}_{s}(Y)`` denotes the group consisting of semi-symplectic automorphisms of ``Y``. 
+The quotient ``\mathrm{Aut}(Y)/\mathrm{Aut}_{s}(Y)`` is a finite group and known to be cyclic in most cases [BG24](@cite).
+
+Let ``\pi \colon X \to Y`` be the K3 cover of ``Y``. 
+
+# Input:
+- `Y` -- represents an Enriques surface in terms of ``\pi^*\colon S_Y(2) \hookrightarrow S_X``.
+- `max_nchambers` -- abort the computation after `max_nchambers` chambers have been computed; return the generators, chambers and curves computed so far. They may not generate the full automorhism group, and not cover all orbits of chambers or rational curves. 
+
+# Output:
+1. A matrix group, the image of ``\mathrm{Aut}_{s}(Y) \to O(S_Y)`` 
+2. A complete list of representatives of the ``G:=G_Y^0``-congruence classes of ``S_Y|S_X``-chambers.
+3. A list of ``(-2)``-vectors representing smooth rational curves on ``Y`` 
+   such that any rational curve of ``Y`` is in the same ``\mathrm{Aut}(Y)``-orbit as at least one class in the list. 
+
+# Examples 
+Let ``Y`` be an Enriques surface with finite automorphism group of type ``I``. Assume that it is very general so that the covering K3 surface ``X`` has picard rank ``19`` and no extra non-symplectic automorphisms.
+In the terminology of [BS24a](@cite) it is an ``(E_8+A_1,E_8+A_1)``-generic Enriques surface, namely the one of Number 172 in Table 1.1. Following Section 7.4 of loc. cit. we compute some invariants. 
+
+The group ``\mathrm{Aut}(Y)`` is a dihedral group of order ``8`` and its image ``\mathrm{Aut}^*(Y)`` in ``O(S_Y)`` is a group of order ``4``. It is this group we compute.
+```jldoctest EnriquesAut
+julia> SY, SX, L26, w, u = load("data/TauTaubarGenericEnriquesSurfaces/TauTaubarGenericEnriquesSurfaceNo172.mrdi");
```

Comment:
If the surfaces are numbered then it probably makes sense to implement a load_enriques_surface function and then users can just pass an int as oppose to trying to find the path similar to the johnson_solid function or similar to the code Oscar/src/AlgebraicGeometry/Surfaces/SurfacesP4.jl .
Then adding a test in the data section to make sure we are still able to load the files when there is an upgrade is also import.

Reply:
Thanks for the suggestion. Done.

Reply:
The test is part of a doctest.

Reply:
Or should I add a test in the data section anyways?

Reply:
(calling the function takes 5 seconds though)

Reply:
If the data between files has exactly the same types then  this should be fine.
Otherwise it would good to make sure that we test all files with different type structure.

Reply:
All exactly the same types.

Reply:
Great, then I think the doc test is sufficient